CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+
+## [4.1.0] 2021-05-26
+Version 4.1.0 of the IDS Information Model
+
+### Added
+
+* `ids:ConfigurationManager` can refer to the configured App Resources, Brokers, App Stores and Clearing Houses represented as catalogs.
+* `ids:AppRoute` can refer to Broker and Clearing House instances (via `ids:appRouteClearingHouse` and `ids:appRouteBroker`), which are used for publishing and logging related communication.
+* `ids:GenericEndpoint` for non-IDS endpoints.
+* Catalog subclasses for Broker, AppStore, AppResource and ClearingHouse instances. Check the [Catalog.ttl](model/infrastructure/Catalog.ttl) class for details.
+* Additional properties for Participants for identification and classification, e.g., legal name, business identifier, legal form and VAT ID (addressing requirements to participants in Gaia-X)
+* Additional properties for `ids:DataApp` and `ids:AppRepresentation` to express supported usage policies as well as app-related runtime, configuration and deployment information.
+
+### Changed:
+* `ids:ParticipantRequestMessage` and `ids:ParticipantResponseMessage` are deprecated and will be removed in the next release. Using `ids:DescriptionRequestMessage` and `ids:DescriptionReponseMessage` is advised.
+
+
+### Removed
+* `idsm:` metamodel annotations, which were used to annotate cardinalities of properties. The property cardinalities and restrictions are now exclusively represented via the corresponding SHACL shapes found in the [testing subdirectory](testing/)
```

Comment:
A global search for `idsm:` yields 77 matches in 11 files. Please double-check and remove these, if possible: 
![image](https://user-images.githubusercontent.com/15930331/121685116-d7062300-cabf-11eb-8b4b-d94e841c74c5.png)

Reply:
Resolved in #460 

Removed all idsm related classes / properties (except for the occurences in the lifting shapes)